### ISSUE TITLE
cross language categories

### ISSUE THREAD

**Issue by elf-pavlik** [shared-stuff/shared-stuff#18]
at this moment person looking for a book may need to search for 'book' (en), 'buch' (de) etc. maybe stuff like _semantic tagging_ could help with referring to for example dbpedia entries?

**Comment by shared-stuff** [comment]
I would propose to provide predefined categories. There will be a button "Add Category" button  (or just a plus icon)  next to the categories which opens a category selector. This ways we can easily provide translations (later) and searching/browsing for stuff will be much easier.

**Comment by elf-pavlik** [comment]
yeah! i remember @jancborchardt suggesting it when we met, we can start by looking at existing items and then make survey of other sharing sites to see what people use there.

**Comment by shared-stuff** [comment]
yes, I don't remember the web site jan mentioned. I would  also look for the categories of big players like ebay or amazone. It's not the same, as sharing is different from selling. But I guess they invested some millions in usability research about their categories. Knowledge we can get  harvest for free.

**Comment by jancborchardt** [comment]
The website I mentioned was [BSR gift market](http://www.bsr.de/verschenkmarkt/list.asp) (by the Berlin trash industry, also available in other German cities). Their categories are:
furniture, home appliances, computer/hifi, telephone, books, hobby, for kids, clothing/shoes, garden/building, other

Amazon.com has:
Books; Movies, Music & Games; Electronics & Computers; Home, Garden & Tools; Grocery, Health & Beauty; Toys, Kids & Baby; Clothing, Shoes & Jewelry; Sports & Outdoors; Automotive & Industrial

I’d propose to use the Amazon categories (good call Marco), with the modification that it’s just »Grocery & Health« and »Clothing & Shoes« (we’re not a company ;D). If we see that we’re missing something we can always add it later.
